pkg/cloud/services/securitygroup/securitygroups.go

```diff
@@ -423,7 +423,7 @@ func (s *Service) revokeAllSecurityGroupIngressRules(id string) error {
 
 	securityGroups, err := s.EC2Client.DescribeSecurityGroups(describeInput)
 	if err != nil {
-		return errors.Wrapf(err, "failed to query security group %q", id)
+		return err
```

Comment:
Hey @pydctw, yes I need to change it because `err` returned by this function is handled by `IsIgnorableSecurityGroupError` function, which expects error to be of `Code` type and if error is not of `Code` type then it returns nil, 
Moreover `error.Wrapf` does not return error as `Code` type which makes this `error = nil` condition at caller function.

Reply:
Thanks for the explanation. Looks good.